frontend/src/components/Header.vue

```diff
@@ -8,6 +8,22 @@
       <div class="divider"></div>
       <span>AWS Power-Toggle</span>
     </div>
+    <div class="bills-container" v-if="totalBillsAccruedLabel && totalBillsSavedLabel">
```

Comment:
I think this is too much for now (could be a user option/setting in the future). Limiting the scope per Environment is enough. In my mind, usually teams would care about a subset of Environments and not all of them.